packages/extensions/openapi-to-cadl/test/batch/cadl-output/models.tsp

```diff
@@ -1,38 +1,41 @@
-import "@cadl-lang/rest";
-import "@azure-tools/cadl-azure-core";
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-core";
 
-using Cadl.Rest;
+using TypeSpec.Rest;
+using TypeSpec.Http;
 using Azure.Core;
 
 namespace Azure.Batch;
 
 @Azure.Core.fixed
 enum OSType {
-  @doc("The Linux operating system.") "linux",
-  @doc("The Windows operating system.") "windows",
+  @doc("The Linux operating system.") linux,
+  @doc("The Windows operating system.") windows,
 }
 @Azure.Core.fixed
 enum VerificationType {
   @doc("""
 The Image is guaranteed to be compatible with the associated Compute Node agent
 SKU and all Batch features have been confirmed to work as expected.
 """)
-  "verified",
+  verified,
   @doc("""
 The associated Compute Node agent SKU should have binary compatibility with the
 Image, but specific functionality has not been verified.
 """)
-  "unverified",
+  unverified,
```

Comment:
will this change output an enum with the same values?

Reply:
It shouldn't change the output. The change comes from recent prettier plugin changes: https://github.com/microsoft/typespec/commit/766cdb728f662c8fdde6417854a301d615ed7cfc#diff-68104c1f8d7943b8b54125fab0d85436006884c0d27ec0cda487d209f9345835